connectd/connectd.c

```diff
@@ -426,8 +545,16 @@ static void PRINTF_FMT(5,6)
 		     err);
 }
 
+/*~ This is the destructor for the (unsuccessful) connection.  We accumulate
+ * the errors which occurred, so we can report to lightningd properly in case
+ * they all fail, and try the next address.
+ *
+ * This is a specialized form of destructor which takes an extra argument;
+ * it set up by either the creatively-named tal_add_destructor2(), or by
+ * the ccan/io-specific io_set_finish() on a connection. */
```

Comment:
`the ccan/io-specific` -> `ccan/io's` ? only because it took me a second to realize it's not `ccan/io-specific`  but `ccan/io`-specific.

Reply:
Good catch!